test/image/mocks/gl2d_parcoords_1.json

```diff
@@ -0,0 +1,29 @@
+{
+  "layout": {
+    "width": 284,
+    "height": 400
+  },
+
+  "data": [{
+
+    "type": "parcoords",
```

Comment:
Something is off with non-default `domain`:

e.g. with `domain.y : [0.5, 1]`

![image](https://cloud.githubusercontent.com/assets/6675409/22747582/6ca6b46c-edf5-11e6-93c8-3c040eefa907.png)

and here with `domain.x = [0.5, 1]`:

![image](https://cloud.githubusercontent.com/assets/6675409/22747619/8c020d8e-edf5-11e6-8a91-f4843635d79a.png)